public/doc/openapi-3-v1.2.json

```diff
@@ -241,13 +241,13 @@
                         }
                     },
                     "400": {
-                        "$ref": "#/components/responses/400BadRequestResponse"
+                        "$ref": "#/components/schemas/400BadRequestResponse"
```

Comment:
@hsong-rh When you have multiple parameters and all of them bad, will that generate a single error element or would it have an array of multiple error elements. And would all of the error elements in the array have a status of 400. We might want to test this use case

Reply:
Only single error element is created with 400 status, even multiple params are bad.